src/main/java/uk/gov/pay/ledger/transaction/model/Dispute.java

```diff
@@ -0,0 +1,215 @@
+package uk.gov.pay.ledger.transaction.model;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import uk.gov.pay.ledger.transaction.search.model.SettlementSummary;
+import uk.gov.pay.ledger.transaction.state.TransactionState;
+
+import java.time.ZonedDateTime;
+
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class Dispute extends Transaction {
+    private final TransactionState state;
+    private final ZonedDateTime createdDate;
+    private final Integer eventCount;
+    private final Long netAmount;
+    private final Long fee;
+    private final String gatewayTransactionId;
+    private final ZonedDateTime evidenceDueDate;
+    private final String reason;
+    private final SettlementSummary settlementSummary;
+    private final Payment paymentDetails;
+    private final String parentTransactionId;
+    private final String transactionId;
+
+    private Dispute(Builder builder) {
+        super(builder.id, builder.gatewayAccountId, builder.serviceId, builder.live, builder.amount,
+                builder.externalId, builder.gatewayPayoutId);
+        this.state = builder.state;
+        this.createdDate = builder.createdDate;
+        this.eventCount = builder.eventCount;
+        this.netAmount = builder.netAmount;
+        this.fee = builder.fee;
+        this.gatewayTransactionId = builder.gatewayTransactionId;
+        this.evidenceDueDate = builder.evidenceDueDate;
+        this.reason = builder.reason;
+        this.settlementSummary = builder.settlementSummary;
+        this.paymentDetails = builder.paymentDetails;
+        this.parentTransactionId = builder.parentTransactionId;
+        this.transactionId = builder.transactionId;
+    }
+
+    public TransactionState getState() {
+        return state;
+    }
+
+    public ZonedDateTime getCreatedDate() {
+        return createdDate;
+    }
+
+    public Integer getEventCount() {
+        return eventCount;
+    }
+
+    public Long getNetAmount() {
+        return netAmount;
+    }
+
+    public Long getFee() {
+        return fee;
+    }
+
+    public String getGatewayTransactionId() {
+        return gatewayTransactionId;
+    }
+
+    public ZonedDateTime getEvidenceDueDate() {
+        return evidenceDueDate;
+    }
+
+    public String getReason() {
+        return reason;
+    }
+
+    public SettlementSummary getSettlementSummary() {
+        return settlementSummary;
+    }
+
+    public Payment getPaymentDetails() {
+        return paymentDetails;
+    }
+
+    public String getParentTransactionId() {
+        return parentTransactionId;
+    }
+
+    public String getTransactionId() {
```

Comment:
It doesn't look like this is used anywhere. We're already setting the `externalId` field on the base Transaction, so I think this field is unnecessary?